packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -588,6 +588,40 @@ export default class TabAdvanced extends Component {
     blockedOperations => STOP_OPERATIONS.every(op => op in blockedOperations)
   )
 
+  _getIsMigrationAllowed = createSelector(
+    () => this.props.vm?.blockedOperations,
+    blockedOperations =>
+      blockedOperations !== undefined && !['migrate_send', 'pool_migrate'].every(op => op in blockedOperations)
```

Comment:
Make sure you consider it blocked only if all the operations are blocked (to make it in-line with the way it works for _stop_.

Reply:
Done